packages/clay-core/src/picker/Picker.tsx

```diff
@@ -382,6 +382,16 @@ export function Picker<T extends Record<string, any> | string | number>({
 		);
 	}
 
+	const getComponentWidth = () => {
+		if (typeof width === 'string' || typeof width === 'number') {
+			return width;
+		}
```

Comment:
I think we always need to maintain the 160px limit regardless of whether the user passes their own value.

Reply:
I think we have that coverage on line 392
```javascript
return clientWidth > 160 ? `${clientWidth}px` : '160px';
``` 

The bug is that we need to allow the user to set the width and make them be readjusted from their content If we add this limit, wouldn't it be an impediment to that?

Reply:
This limit is only minimum so if the size is greater than that it is ok.

Reply:
The initial requirements for this allowed for a width smaller than 160px such as in the case of the date picker for years. We shouldn't prevent a smaller width for that type of case.

Reply:
That's true, I ended up forgetting about the DatePicker use case.